Main/Main.ino

```diff
@@ -1,7 +1,6 @@
 /* 
  *  
- *  Creator : Thomas Cibils
- *  Last update : 05.02.2019
+ *  Creator : Thomas Cibils & Matteo Raviglione
```

Comment:
Is this change needed by the client? Can you please link the JIRA ticket?
Thanks so much,

Reply:
xD